### PR TITLE
Only pom file for master branch

### PR DESCRIPTION
**Pull request by YatKat**

### CODE REVIEW

File: src/main/java/mate/academy/jpademo/dao/CRUDDao.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package mate.academy.jpademo.dao;
+
+import java.util.List;
+
+public interface CRUDDao<T, ID> {
```
Comment (ghost) [inline src/main/java/mate/academy/jpademo/dao/CRUDDao.java]: should the name of a class be uppercase?

File: src/main/java/mate/academy/jpademo/dao/impl/AccessoryDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+public class AccessoryDaoImpl extends AbstractDao implements AccessoryDao {
+
+    public AccessoryDaoImpl(SessionFactory sessionFactory) {
+        super(sessionFactory);
+    }
+
+    @Override
+    public Accessory save(Accessory entity) {
+        Session session = sessionFactory.openSession();
+        session.save(entity);
```
Comment (ghost) [inline src/main/java/mate/academy/jpademo/dao/impl/AccessoryDaoImpl.java:20]: I'm not sure whether we should close the session in a session factory or not, check it pls

File: pom.xml [hunk 3]
```diff
@@ -62,6 +62,12 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (ghost) [inline pom.xml:65]: delete this one from git cache

File: src/main/java/mate/academy/HibernateMain.java [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        patient.setLastName("Pupkin");
+        patient.setDateOfBirth(new Date(1996, 05, 02));
+        ServicePatient servicePatient = new ServicePatientImpl
+                (new PatientDaoImpl(HibernateUtil.getSessionFactory()));
+        servicePatient.save(patient);
+        return patient;
+    }
+
+    public static Patient doTest (Patient patient, Test test){
+        ServiceTest serviceTest = new ServiceTestImpl
```
Comment (ghost) [inline src/main/java/mate/academy/HibernateMain.java:44]: you could use dependency container for service layer

File: src/main/java/mate/academy/jpademo/controller/PatientInfoServlet.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package mate.academy.jpademo.controller;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class PatientInfoServlet extends HttpServlet {
```
Comment (ghost) [inline src/main/java/mate/academy/jpademo/controller/PatientInfoServlet.java:9]: we dont have this in task

File: src/test/java/mate/academy/RegExpTest.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+public class RegExpTest {
+
+    @Test
+    public void mobilePhoneValidation(){
```
Comment (ghost) [inline src/test/java/mate/academy/RegExpTest.java]: we don't have this in task

**Review by ghost — COMMENTED** [review]
LGTM

**Comment by ghost** [comment]
LGTM

File: src/main/java/mate/academy/jpademo/model/devices/Device.java [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+    private Integer id;
+    @Column
+    private String name;
+    @Column
+    private String model;
+    @Column
+    private String brand;
+
+    @ManyToMany(fetch = FetchType.LAZY, cascade = CascadeType.ALL, mappedBy = "deviceSet")
+//    @JoinTable(name = "devices_accessory", joinColumns = {@JoinColumn(name = "device_id")},
```
Comment (boroda4436) [inline src/main/java/mate/academy/jpademo/model/devices/Device.java]: remove commented code

File: src/main/resources/hibernate.cfg.xml [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+        <property name="hibernate.show_sql">true</property>
+        <property name="hibernate.hbm2ddl.auto">update</property>
+        <property name="hibernate.cache.region.factory_class">org.hibernate.cache.ehcache.EhCacheRegionFactory</property>
+        <property name="hibernate.cache.use_second_level_cache">true</property>
+        <property name="hibernate.c3p0.min_size">5</property>
+        <property name="hibernate.c3p0.max_size">20</property>
+        <property name="hibernate.c3p0.timeout">300</property>
+        <property name="hibernate.c3p0.max_statements">50</property>
+        <property name="hibernate.c3p0.idle_test_period">3000</property>
+        <mapping class="mate.academy.jpademo.model.devices.Device"/>
```
Comment (boroda4436) [inline src/main/resources/hibernate.cfg.xml]: You may remove mapping class. You already have declared classes in HibernateUtil class

**Review by boroda4436 — CHANGES_REQUESTED** [review]

**Review by boroda4436 — APPROVED** [review]
LGTM

**Pull request merged**